app/src/main/java/org/kinecosystem/kinit/view/TabsAdapter.kt

```diff
@@ -9,39 +9,36 @@ import android.view.ViewGroup
 import org.kinecosystem.kinit.KinitApplication
 import org.kinecosystem.kinit.R
 import org.kinecosystem.kinit.analytics.Analytics
-import org.kinecosystem.kinit.databinding.*
+import org.kinecosystem.kinit.databinding.BalanceTabBinding
+import org.kinecosystem.kinit.databinding.EarnTabBinding
+import org.kinecosystem.kinit.databinding.InfoTabBinding
+import org.kinecosystem.kinit.databinding.SpendTabBinding
 import org.kinecosystem.kinit.navigation.Navigator
 import org.kinecosystem.kinit.network.TaskService
 import org.kinecosystem.kinit.network.Wallet
 import org.kinecosystem.kinit.repository.TasksRepository
 import org.kinecosystem.kinit.util.Scheduler
-import org.kinecosystem.kinit.view.BottomTabNavigation.PageSelectionListener
 import org.kinecosystem.kinit.view.adapter.BalancePagerViewsAdapter
 import org.kinecosystem.kinit.view.adapter.OfferListAdapter
-import org.kinecosystem.kinit.view.customView.LockableViewPager
 import org.kinecosystem.kinit.viewmodel.balance.BalanceViewModel
 import org.kinecosystem.kinit.viewmodel.earn.EarnViewModel
 import org.kinecosystem.kinit.viewmodel.info.InfoViewModel
 import org.kinecosystem.kinit.viewmodel.spend.SpendViewModel
 import javax.inject.Inject
 
-private const val NUMBER_OF_TABS = 5
-private const val PRE_EARN_SHOW_DURATION: Long = 2000L
+private const val NUMBER_OF_TABS = 4
```

Comment:
should be static right? -> move into companion object

Reply:
It's a private **local** constant *value*, not a public property

Reply:
i'm not saying to set is a public, you can just write:
`private const val NUMBER_OF_TABS = 4` inside companion object and we make it static like that.